### ISSUE TITLE
YTPReady event firing up non-stop after switching to another video

### ISSUE THREAD

**Issue by ncla** [pupunzi/jquery.mb.YTPlayer#162]
This issue happens on Firefox 41.0.1 and Chrome 49.0.2623.28 beta-m (64-bit).

I can't exactly pin point why this is happening. Initially I thought it had something to do with the volume adjusting, but without it, the issue still occurs. It is a bit number as well, sometimes it does happen, sometimes it doesn't. Doesn't seem to be a specific video to cause this.

I recently upgraded to newer version because I saw you resolved some volume issue (which it did resolve for me as well). The version I have is 2.9.11. I don't recall this event issue happening before I updated this plugin.

The event does not fire however if you switch tabs, but it does keep continue playing audio.

Code attached below, you can also visit http://ncla.me and see the problem there as well.

```
var videos = [
    {
        videoURL: '5jHFLa0XB3c',
        containment: 'body',
        autoPlay: true,
        startAt: 225,
        stopAt: 400,
        opacity: 1,
        loop: false,
        ratio: "16/9",
        addRaster: true,
        showControls: true,
        mute: true,
        vol: 50
    },
    {
        videoURL: 'LhYYd5adVY4',
        containment: 'body',
        autoPlay: true,
        startAt: 120,
        opacity: 1,
        loop: false,
        ratio: "16/9",
        addRaster: true,
        showControls: true,
        mute: true,
        vol: 50
    },
    {
        videoURL: 'EF_xdvn52As',
        containment: 'body',
        autoPlay: true,
        startAt: 90,
        opacity: 1,
        loop: false,
        ratio: "16/9",
        addRaster: true,
        showControls: true,
        mute: true,
        vol: 50
    },
    {
        videoURL: 'UWOEFkauoYY',
        containment: 'body',
        autoPlay: true,
        startAt: 11,
        stopAt: 100,
        opacity: 1,
        loop: false,
        ratio: "16/9",
        addRaster: true,
        showControls: true,
        mute: true,
        vol: 50
    },
    {
        videoURL: 'VXPoJAyeF8k',
        containment: 'body',
        autoPlay: true,
        startAt: 145,
        stopAt: 270,
        opacity: 1,
        loop: false,
        ratio: "1.85/1",
        addRaster: true,
        showControls: true,
        mute: true,
        vol: 50
    },
    {
        videoURL: 'g7d9KM1plyg',
        containment: 'body',
        autoPlay: true,
        startAt: 85,
        stopAt: 165,
        opacity: 1,
        loop: false,
        ratio: "16/9",
        addRaster: true,
        showControls: true,
        mute: true,
        vol: 50
    }
];

var unmuted = false;
var count = 0;
var init = false;

function setPlayerVolume(vol) {
    console.log('Chaning volume to', vol);
    $('#player').YTPSetVolume(vol);

    unmuted = (vol > 0);
}

function checkKey(e) {
    e = e || window.event;

    if (e.keyCode == '38') {
        count++;

        if (count > 5 && !unmuted) {
            $("#player").YTPUnmute();

            setPlayerVolume(50);

            unmuted = true;
        }
    }
    else if (e.keyCode == '37') {
        // left arrow
        $('#player').YTPPlayPrev();
    }
    else if (e.keyCode == '39') {
        // right arrow
        $('#player').YTPPlayNext();
    }
}

$(document).ready(function() {
        $(function() {
            $("#player").YTPlaylist(videos, true);

            var player = $("#player").YTPlayer();

            player.on('YTPReady', function(e) {
                console.error('READY 1');
                if (!init) {
                    init = true;
                    console.error('READY 2');
                    $('#home').addClass('show');
                    $('body').addClass('video-loaded');
                }

                if (unmuted) {
                    setPlayerVolume(50);
                }
            });
        });
    });
```

**Comment by pupunzi** [comment]
Hi Ncla,
just fixed this issue check the 2.9.13 version.
Bye,
Matteo

**Comment by ncla** [comment]
:+1: 

Thanks, from quick testing it seems to be fixed.
